src/client-api-proxy.ts

```diff
@@ -22,6 +23,22 @@ export async function getStories(
 ): Promise<BrazeContentProxyResponse> {
   let data: ClientApiResponse | null = null;
 
+  // Validate inputs
```

Comment:
refactor the validation to a single function that calls the two separate validation functions